justfile

```diff
@@ -70,7 +71,10 @@ cargo-features := trim_end_match(if override-features != "" { override-features
     } else { extra-features
 }, ",")
 
-cargo-build-args := (if for-release != "" { " --release" } else { "" }) + (if ci != "" { " --locked" } else { "" }) + (if target != target-host { " --target " + target } else if cargo-buildstd != "" { " --target " + target } else { "" }) + (cargo-buildstd) + (if extra-build-args != "" { " " + extra-build-args } else { "" }) + (cargo-no-default-features) + (if cargo-features != "" { " --features " + cargo-features } else { "" })
+cargo-split-debuginfo := if for-release != "" { " --config='profile.release.split-debuginfo=\"packed\"' --config=profile.release.debug=2" } else { "" }
```

Comment:
P.S. packed is for win and macos only.
For other os we need to use unpacked.

Reply:
No, linux has packed too: https://blog.rust-lang.org/2022/11/03/Rust-1.65.0.html#splitting-linux-debuginfo

Reply:
Thanks, didn't know that.
Looks like the doc for unstable feature is oudated.